cspell.yaml

```diff
@@ -53,4 +54,3 @@ words:
   - vmss
   - vnet
   - westus
-caseSensitive: false
```

Comment:
@richardpark-msft: Why remove this?

Reply:
Hm. That was not on purpose, I'll revert that segment.

Reply:
Fixed.